README.md

```diff
@@ -31,80 +31,3 @@ Ubuntu 20 this is ROS Noetic.
 - Catkin
 
 *+ package specific dependecies*
-
-
-## Installation 
-
-To Install ROS (desktop-full version) follow instructions on the ROS [wiki](http://wiki.ros.org/noetic/Installation/Ubuntu).
-
-We recommend using the catkin command line tools as opposed to whatever is bundled with ROS:
-
-    sudo apt-get install python3-catkin-tools
-
-To get these project files onto your computer using git navigate to a desired folder and do:
-
-    git clone https://github.com/mcgill-robotics/AUV-2024.git
-  
-Alternatively, if you set up your SSH keys with your Github account you can save yourself having to type your user/pass every 
-time (**recommended**):
-
-    git clone git@github.com:mcgill-robotics/AUV-2024.git
-
-
-## Building ROS Packages
-
-Before building packages, make sure that the following additional ROS/python packages are installed:
-
-	sudo apt-get install ros-noetic-rosserial-arduino
-	sudo apt-get install ros-noetic-pid
-	sudo apt-get install ros-noetic-joy
-	sudo apt-get install ros-noetic-joy-teleop
-	sudo apt-get install ros-noetic-sbg-driver
-	sudo apt-get install ros-noetic-usb-cam
-	sudo apt-get install ros-noetic-realsense2-camera
-	pip install numpy-quaternion
-	pip install ultralytics
-
-*To build/launch/test an individual package, see the README.md in the respective package.*
-
-To build the currently functional packages:
-
-	source /opt/ros/noetic/setup.bash
-	cd <AUV-2020>/catkin_ws
-	catkin build bringup
-
-After build is complete, update ROS environment so that packages are 'visible':
-
-	source ./devel/setup.bash
-  
-  
-### Flashing Firmware
-
-If you have an Arduino you can wire it up to emmulate the thrusters on the AUV. Upload the 
-binary file by connecting the Arduino via USB (currently hardcoded assumption USB shows up as /dev/ttyACM0) and running 
-the generated make target:
-
-    catkin build --no-deps  propulsion --make-args propulsion_embedded_thrusters-upload
-    catkin build --no-deps  depth_sensor --make-args depth_sensor_embedded_depth_sensor-upload
-    catkin build --no-deps  imu --make-args imu_embedded_imu-upload
```

Comment:
can you add the flashing commands to the wiki

Reply:
could you also link to the Wiki to read about installation/usage

Reply:
does anyone actually use this? I thought it was just old stuff